src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -313,6 +313,11 @@ public static FormulaValue Upper(IRContext irContext, StringValue[] args)
             return new StringValue(irContext, args[0].Value.ToUpper());
         }
 
+        public static FormulaValue Proper(IRContext irContext, StringValue[] args)
+        {
+            return new StringValue(irContext, CultureInfo.CurrentCulture.TextInfo.ToTitleCase(args[0].Value));
```

Comment:
JS implementation was done with regex + using the current locale. Check in powerapps that this handles stuff like acronyms the same way. 
![image](https://user-images.githubusercontent.com/69215460/165844771-60ace210-0da9-4062-91d5-0dcd24e4340c.png)

Reply:
![image](https://user-images.githubusercontent.com/23180943/165857183-0fc8693b-4761-49bf-9bcc-de526f967700.png)
Not sure if i fully understand you. Did you mean something like this? Are we expecting "RSVP" to maintain as "RSPV" after the proper function? Let me know if i set this up wrong

Reply:
I meant that the ToTitleCase function in C# would keep it as RSVP according to the docs https://docs.microsoft.com/en-us/dotnet/api/system.globalization.textinfo.totitlecase?view=net-6.0

So we have a difference in behavior here.

Reply:
I made the change, let me know if you think this is ok